frontend/src/views/Pages/Register.vue

```diff
@@ -160,6 +174,8 @@ export default {
       checkPassword: '',
       passwordVisible: false,
       submitted: false,
+      registerfail: false,
```

Comment:
Can we rename this field into something more meaningful like `showError` or `errorAlertvisible` please?

Reply:
erledigt

Reply:
Lügen!